ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoDivulgenceSpec.scala

```diff
@@ -56,6 +56,7 @@ private[dao] trait JdbcLedgerDaoDivulgenceSpec extends LoneElement with Inside {
       val builder = TransactionBuilder()
       val rootExercise = builder.add(
         NodeExercises(
+          observers = Set.empty,
```

Comment:
Given that this test is about divulgence and choice observers are too, we'll need some new tests here.